posthog/urls.py

```diff
@@ -265,6 +291,7 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> st
     # auth
     path("logout", logout, name="login"),
     path("login", login_view, name="login"),
+    path("signup/finish/", finish_social_signup, name="signup_finish"),
```

Comment:
This reminded me, what happens if the signup is not completed? Should we have some cleanup task to make sure we don't keep unnecessary PII?

Reply:
Well, we keep the session in the DB (table `social_auth_partial`) in case the user returns, which we could clean periodically.

Reply:
Added a Celery task to remove sessions older than 7 days every Friday.